src/norm/pragmas.nim

```diff
@@ -19,3 +19,80 @@ template ro* {.pragma.}
 
 template unique* {.pragma.}
   ## Mark field as unique.
+
+template fk*(val: typed) {.pragma.}
+  ##[ Mark ``int`` field as foreign key. Foreign keys always references the field ``id`` of ``val``. ``val`` should be a Model.
+  ]##
+runnableExamples:
```

Comment:
I used to use runnableExamples in the docstring but they turned to be too fragile. If this exact code is already in the tests, we can safely remove it from here. But I'm open to a discussion on, I can see how they are useful.

Reply:
IMO, the benefit of runnableExample is that they are compiled & executed when you use ``nim doc``.

This allows for: 
* Knowing if your examples are not correct (catches typo, things like that).
* Old obsolete example that do not work anymore do not get forgotten. Make maintenance easier.
* You can use ``nim doc`` as part of your validation process (for instance, by having a tests that execute nim doc) to make cover more case (without having to write both a test and an example).

That said, if you don't use nim doc I can just remove it.